test/acceptance/pages/ApplicationPage.scala

```diff
@@ -38,4 +38,11 @@ object ApplicationPage extends WebPage {
     click on blockApplicationButton
   }
 
+  def unblockApplicationButton = find(id("unblock-application")).get
+
+  def selectUnblockApplication() = {
+    click on unblockApplicationButton
+  }
+
+
 }
```

Comment:
Maybe delete this file and use the selectUnblockApplication from ApplicationPage.scala